testpilot/frontend/static-src/scripts/legal.js

```diff
@@ -0,0 +1,62 @@
+window.addEventListener('DOMContentLoaded', function (event) {
+  'use strict';
+
+  // Add an accordion to the Privacy Notice page
+  if (location.pathname === '/privacy') {
+    var ps = document.querySelectorAll('h2 ~ p');
+    var tablist = ps[0].parentElement;
```

Comment:
I don't know if it helps, but we currently "own" the Privacy Notice and other legal docs in [./legal-copy](https://github.com/mozilla/testpilot/tree/master/legal-copy), so we could probably inject some HTML with classnames/IDs into the Markdown, if that makes selectors and DOM grabbing any easier.